src/solvers/dgsem_unstructured/sort_boundary_conditions.jl

```diff
@@ -13,9 +13,10 @@ It stores a set of global indices for each boundary condition type and name to e
 during the call to `calc_boundary_flux!`. The original dictionary form of the boundary conditions
 set by the user in the elixir file is also stored for printing.
 """
-mutable struct UnstructuredSortedBoundaryTypes{N, BCs <: NTuple{N, Any}}
+mutable struct UnstructuredSortedBoundaryTypes{N, BCs <: NTuple{N, Any},
+                                               Vec <: AbstractVector{<:Integer}}
     boundary_condition_types::BCs # specific boundary condition type(s), e.g. BoundaryConditionDirichlet
-    boundary_indices::NTuple{N, Vector{Int}} # integer vectors containing global boundary indices
+    boundary_indices::NTuple{N, Vec} # integer vectors containing global boundary indices
     boundary_dictionary::Dict{Symbol, Any} # boundary conditions as set by the user in the elixir file
     boundary_symbol_indices::Dict{Symbol, Vector{Int}} # integer vectors containing global boundary indices per boundary identifier
 end
```

Comment:
Aqua.jl complains about unbound type parameters here, see https://github.com/trixi-framework/Trixi.jl/actions/runs/13117124962/job/36594056579?pr=2212#step:7:6391

Reply:
Ja, das verstehe ich nicht ganz.

Reply:
This looks like https://github.com/JuliaTesting/Aqua.jl/issues/86.

Reply:
See also https://github.com/JuliaTesting/Aqua.jl/issues/265#issuecomment-2173168334, which is basically exactly the case here.

Reply:
If there is no way, you can set `unbound_args = false` for Aqua.jl tests so that we can proceed.